src/main/java/com/example/booksstore/exceptions/CustomGlobalExceptionHandler.java

```diff
@@ -48,6 +48,11 @@ private String getErrorMessage(ObjectError e) {
         return e.getDefaultMessage();
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    public ResponseEntity<Object> handleRegistrationException(RegistrationException exception) {
+        return new ResponseEntity<>(exception.getMessage(), HttpStatus.CONFLICT);
+    }
```

Comment:
I understand you, but if I will try to make 1 method from 2 exception methouds, I can't handle 1 exception message and get it. Because in this case i should to add exception to parameters like this
![image](https://github.com/Recdt/Books-Store/assets/90136583/ad46c42f-7f4f-41f6-b0b9-c5a3924d6636)
and I think that I can't get 2 exceptions in 1 method like parameters